install/operator/pkg/cmd/internal/install/install_forge.go

```diff
@@ -26,7 +26,7 @@ import (
 	"unicode"
 
 	"github.com/spf13/cast"
```

Comment:
General note: really want to remove install-forge and olm gen from syndesis binary. Neither are used.

Reply:
Good, I will do this in another PR.